tests/accelerators/test_ddp.py

```diff
@@ -83,7 +82,7 @@ def test_cli_to_pass(tmpdir, args=None):
     return '1'
 
 
-@pytest.mark.skipif(platform.system() == "Windows", reason="Distributed training is not supported on Windows")
+@RunIf(skip_windows=True)
```

Comment:
I find this `RunIf` decorator confusing as you provide arguments to Skip.

Reply:
We could simply provide both RunIf and SkipIf. Sometimes it makes sense to use one or the other.

Reply:
yes, but then is miss the beauty of having a one-line configuration...